core/src/main/kotlin/configuration.kt

```diff
@@ -50,18 +53,18 @@ enum class Platform(val key: String) {
 data class DokkaSourceSetID(
     val moduleName: String,
     val sourceSetName: String
-) : Serializable {
+) {
     override fun toString(): String {
         return "$moduleName/$sourceSetName"
     }
 }
 
 fun DokkaConfigurationImpl(json: String): DokkaConfigurationImpl {
-    return Gson().fromJson(json, DokkaConfigurationImpl::class.java)
+    return parseJson(json)
 }
 
-fun DokkaConfiguration.toJson(): String {
-    return Gson().toJson(this)
+fun DokkaConfiguration.toJsonString(): String {
+    return toJsonString(this)
 }
```

Comment:
I'd suggest to also make it oneline with `=`

Reply:
Done ✅